src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetVerbCommand.cs

```diff
@@ -70,11 +70,14 @@ protected override void ProcessRecord()
                             }
                         }
 
-                        VerbInfo verb = new();
-                        verb.Verb = field.Name;
-                        verb.AliasPrefix = VerbAliasPrefixes.GetVerbAliasPrefix(field.Name);
-                        verb.Group = groupName;
-                        verb.Description = VerbDescriptions.GetVerbDescription(field.Name);
+                        var verb = new VerbInfo()
+                        {
+                            Verb = field.Name,
+                            AliasPrefix = VerbAliasPrefixes.GetVerbAliasPrefix(field.Name),
+                            Group = groupName,
+                            Description = VerbDescriptions.GetVerbDescription(field.Name)
+                        };
+
```

Comment:
I think most of this can be removed since it was moved to the `FilterByVerbsAndGroups` method which creates verb info for argument completer and command in this PR: https://github.com/PowerShell/PowerShell/pull/20286

Probably also explains the merge conflicts now 😄